packages/react/src/components/ButtonGroup/__stories__/ButtonGroup.stories.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withKnobs, object } from '@storybook/addon-knobs';
+import '../../../../../styles/scss/components/buttongroup/_buttongroup.scss';
+import 'carbon-components/scss/globals/grid/_grid.scss';
```

Comment:
I see you're pulling in the grid, but it doesn't appear as if you're using it.

Reply:
Good catch, I pulled this from another story and didn't remove that.